src/spouts/readHygiene.py

```diff
@@ -0,0 +1,32 @@
+from collections import Counter
+from pymongo import MongoClient
+from streamparse import Spout
+import json
+
+class readHygiene(Spout):
+    outputs = ['rec']
+
+    def initialize(self, storm_conf, context):
+        self.client = MongoClient('mongodb://localhost:27017/')
+        self.db = self.client['weisheng']
+        self.aggregatedCollection = 'hygiene'
+        self.counter = 0
+        self.total = self.db[self.aggregatedCollection].find().count()
+
+    def _increment(self):
+        if self.counter>self.total:
+            self.counter = 0
+        self.counter += 1
+
+    def fail(self, tup_id):
+        self.logger.error("Error: "+tup_id)
+
+    def next_tuple(self):
+        self.logger.info(str(self.counter))
+        if(self.counter<self.total):
+            self.rec = self.db[self.aggregatedCollection].find({}, {"_id": 0})[self.counter]
+            self._increment()
+        self.recordJson = json.dumps(self.rec, ensure_ascii=False)
```

Comment:
Sometimes json.loads throws errors due to bad formatting, should put this in a try block